themes/bootstrap3/js/record.js

```diff
@@ -24,7 +24,7 @@ function checkRequestIsValid(element, requestType) {
       if (response.data.status) {
         $(element).removeClass('disabled')
           .attr('title', response.data.msg)
-          .html('<i class="fa fa-flag" aria-hidden="true"></i>&nbsp;' + response.data.msg);
+          .html(VuFind.icon('hold') + response.data.msg + "!!");
```

Comment:
What's the `"!!"` here?

Reply:
The icon should be 'place-request' since it's used for all request types, not just holds. Or optimally, include the icon as a parameter to checkRequestIsValid and add their own icons (e.g. 'place-hold', 'place-storage-request', 'place-ill-request').